utils/models.py

```
@@ -0,0 +1,178 @@
+    ):
+        tokenizer_args = {
+            "max_length": self.model_max_length,
+            "truncation": truncate_tokens,
+        }
+
+        if return_tensors:
+            tokenizer_args["return_tensors"] = return_tensors
+
+        return self.tokenizer(article_body, **tokenizer_args)
+
+    def get_output(self, tokens):
```

Ruff

Missing return type annotation for public function `get_output`

(ANN201)